src/_angular/file-selector/dtv-file-selector.js

```diff
@@ -121,6 +121,23 @@
             return name;
           }
 
+          function showWarnings(type) {
+            scope.folderIsEmpty = false;
+            if (type === "single-folder") {
+              fetch("https://storage-dot-rvaserver2.appspot.com/_ah/api/storage/v0.01/files?companyId=" + scope.companyId + "&folder=" + encodeURI(scope.selector.storageName))
+              .then(function(response) {
+                return response.json()
+              })
+              .then(function(data) {
+                // Note: the first item in the data.files is the folder itself
```

Comment:
@olegrise Although this works from your validation, I'm wondering if it's best to port the logic from `rise-storage` for consistency, see [here](https://github.com/Rise-Vision/rise-storage/blob/master/rise-storage.html#L1446-L1449). I'm okay with not doing that, but wanted to mention since it's been code existing for a long time that definitely works.

Reply:
@stulees I used logic from `rise-storage` and confirmed it's working. See my latest commit.